src/h3lib/lib/polygon.c

```diff
@@ -13,8 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/** @file polygon.c
- * @brief Polygon algorithms
+/** @file polygonGeofence.c
+ * @brief Polygon (Geofence) algorithms
```

Comment:
I can't comment on the right line, but we should probably include 
```
#undef TYPE
#undef IS_EMPTY
#undef INIT_ITERATION
#undef ITERATE
```
after the include